src/bicep/mlz.json

```diff
@@ -5,7 +5,7 @@
     "_generator": {
       "name": "bicep",
       "version": "0.4.1008.15138",
-      "templateHash": "17151661725468237381"
+      "templateHash": "15169166681509362530"
```

Comment:
Exclude this file from the PR because it appears to be only an update to the template hash.

Reply:
Removed.